libs/@guardian/core-web-vitals/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @guardian/core-web-vitals
 
+## 11.0.0
+
+### Patch Changes
```

Comment:
There's a bug in the changesets setup where the changelog says patch here, even though it's a major bump